### PR TITLE
[WIP] Add missing translations and fix typos (pt)

### PR DESCRIPTION
**Pull request by alanwillms**
This PR:

* Add pages that were missing in Portuguese
* Update text according to the [Portuguese language orthographic agreement](https://www.wikiwand.com/en/Portuguese_Language_Orthographic_Agreement_of_1990)
* Add a lot of sections that were missing or outdated.
* Fix typos

This PR doesn't cover blog posts. I will translate and submit them later.

The `/pt` section of the website follows “Portuguese from Portugal” (**pt-PT**), however I am Brazilian (**pt-BR**). There is an international agreement unifying Portuguese language, but a few therms are still different.

Can I keep my changes on the `pt` files or should I create new ones for `pt-BR`?

### CODE REVIEW

**Comment by stomar** [comment]
First, thank you very much for this great effort and systematic work.

But unfortunately, **it's not possible to merge this PR**.

It is based on the state of this repository from nearly a year ago (1c3efa1f4eb0785e2a2580a05f8419fc479dd5b5, 2014-08-20), and your changes are in conflict with many additions and improvements that have been committed since then. (For instance, the `about` section already has been translated.)

What you need to do:
1. Get your master branch in sync with upstream (this **must** be a fast-forward, no merge)
2. Reapply your changes, respecting the changes that others have already made in the meantime. You can inspect the relevant modifications e.g. with `gd --stat 1c3efa1f -- <path>`.

Generally, it's a good idea to start contributing with smaller PR's to get early feedback.

**Comment by stomar** [comment]
Regarding pt_BR: we do not have enough resources to maintain 2 versions of the site, so we agreed on keeping the language as "neutral" as possible, and in case of differences use the pt version.

File: _config.yml [hunk 1]
```diff
@@ -230,8 +230,8 @@ locales:
         url: /pt/community
       - text: Notícias
         url: /pt/news
-#     - text:
-#       url: /pt/security
+      # - text: Segurança
```
Comment (stomar) [inline _config.yml]: remove comments when the page is translated

File: _config.yml [hunk 2]
```diff
@@ -1837,6 +1838,9 @@ locales:
       Proszę udziel nam wsparcia na <a href="https://github.com/ruby/www.ruby-lang.org/">GitHub</a>
       lub skontaktuj się z naszym <a href="mailto:webmaster@ruby-lang.org">webmasterem</a>
       jeśli masz jakieś pytania lub komentarze dotyczące tego serwisu.
+    pt: |
+      <a href="/en/about/website/">Este website</a>
```
Comment (stomar) [inline _config.yml]: en -> pt

File: pt/index.html [hunk 3]
```diff
@@ -16,11 +16,24 @@
...
       </p>
 
-      <a href="downloads/" class="download-link">Download Ruby</a>
+      <a href="downloads/" class="download-link">Baixe o Ruby</a>
       ou
-      <a href="about/">Ler Mais...</a>
+      <a href="about/">Leia Mais...</a>
+
     </div>
-    <div id="code"></div>
```
Comment (stomar) [inline pt/index.html]: noscript code is currently only provided in `en` version.

File: pt/security/index.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+distribuidores, e plataformas PaaS). Os membros precisam ser
+pessoas individuais, outras listas de e-mail não são permitidas.
+
+## Problemas conhecidos
+
+Estes são os problemas recentes:
+
+{% include security_posts.html %}
+
+Outros problemas conhecidos:
```
Comment (stomar) [inline pt/security/index.md]: This list to untranslated posts should be dropped and replaced with:

  `See [the English page](/en/security/) for prior security related posts.` (translated)

**Comment by alanwillms** [comment]
@stomar it seems when you try to refork a repository, GitHub seems to create a new fork, but in fact it just keep the old outdated one. That's what happened.

**Comment by alanwillms** [comment]
Now I guess everything is alright.

File: pt/security/index.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+## Problemas conhecidos
+
+Estes são os problemas recentes:
+
+{% include security_posts.html %}
+
+Consulte a [página em inglês](/en/security/) para os posts relacionados
+à segurança mais antigos.
+
+Outros problemas conhecidos:
```
Comment (stomar) [inline pt/security/index.md]: This list should be removed... see e.g. `it`.

Comment (alanwillms) [inline pt/security/index.md]: Ok, *now* I got it.

File: pt/downloads/index.md [hunk 6]
```diff
@@ -1,76 +1,74 @@
...
 [license]: {{ site.license.url }}
-[installation]: /en/documentation/installation/
+[installation]: /pt/documentation/installation/
 [stable-gz]: {{ site.downloads.stable.url.gz }}
 [previous-gz]: {{ site.downloads.previous.url.gz }}
 [previous20-gz]: {{ site.downloads.previous20.url.gz }}
 [stable-snapshot-gz]: {{ site.downloads.stable_snapshot.url.gz }}
 [nightly-gz]: {{ site.downloads.nightly_snapshot.url.gz }}
-[mirrors]: /en/downloads/mirrors/
+[mirrors]: /pt/downloads/mirrors/
```
Comment (stomar) [inline pt/downloads/index.md]: pt -> en (mirrors page is not to be translated)

**Comment by stomar** [comment]
the installation page is in the wrong location (-> `pt/documentation/installation/`)

**Comment by stomar** [comment]
I'll review more closely soon, no time right now.

**Comment by stomar** [comment]
Sorry for the delay. I have been rather busy, but will sort this out these days.

**Comment by alanwillms** [comment]
No problem, take your time

**Comment by stomar** [comment]
Closed in favor of #1199.

**Pull request closed without merging**